src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,103 @@
+package tasks;
+
+/**
+ * Represents a general task in the task management application.
+ * This class serves as a base for specific types of tasks like ToDo, Deadline, and Event.
+ */
+public class Task {
+    protected String description; // The description of the task
+    protected boolean isDone; //The completion status of the task
+
+    /**
+     * Constructs a Task with the specified description.
+     *
+     * @param description The description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns the description of the task.
+     *
+     * @return The task's description.
+     */
+    public String getDescription() {
```

Comment:
Good naming convention to represent method, it is very intuitive for other developer to understand the purpose of this function.